api/tests/core/providers/test_api.py

```diff
@@ -156,7 +156,7 @@ def test_change_venue_provider():
 
 class SynchronizeStocksTest:
     @pytest.mark.usefixtures("db_session")
-    @freeze_time("2020-10-15 09:00:00")
+    @freeze_time("2022-10-15 09:00:00")
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Same question :) Pourquoi on a besoin de changer ce test ?

Reply:
same answer: J'ai changé les factories c'est une des consequences de ce changement (toutes les modification sur les tests sont des conséquences de la modification des factories)